integration-tests/cypress/e2e/project/portals/portals.cy.ts

```diff
@@ -19,6 +19,20 @@ import { Portal } from '../../../workbench/Portal';
 
 const projectName = 'Cypress - portal';
 
+function checkRepresentationsView(expectedItems: string[]): void {
+  cy.getByTestId('viewselector-Representations').click();
+  cy.getByTestId('view-Representations').then((res) => {
+    const view = cy.wrap(res);
```

Comment:
It's quite odd that this code does not crashes the build, I'll fix it

Reply:
I'm pretty sure that you have something missing in your development environment since there are quite a few additional errors in this file.